docs/articles/old-guide/chatter.py

```diff
@@ -81,20 +84,20 @@ def get_chat_session(conn, channelname):
     # the key 'chat_sessions'.
     root = conn.root()
     if not root.has_key('chat_sessions'):
-        print 'Creating chat_sessions B-tree'
+        print 'Creating chat_sessions B-tree'  # NOQA: E999 print statement
```

Comment:
```suggestion
        print('Creating chat_sessions B-tree')
```

This will even work in Python 2.7.